### ISSUE TITLE
Examples are included in language stats

### ISSUE THREAD

**Issue by RodentOfUnusualSize** [RodentOfUnusualSize/random-message-loader#26]
As of commit 078780626e7da4ca3fc77f16f8f8913d6ce615dd, the repo’s languages stats say:

| Language | Stats |
|----------------|---------|
| JavaScript | 65.6% |
| HTML | 24.5% | 
| Shell | 6.9% |
| CSS | 3.0% |

The entirety of the program is in JavaScript. There are some shell scripts for the testing stuff; maybe all the testing stuff should be marked as not detectable for the language stats, but that’s for another issue.

The issue here is the HTML and CSS, both of which are only used for the examples. Since they are documentation, they should not be included in the language stats.

One way to mark them as documentation would be to create a `.gitattributes` files, and add the line:

```gitattributes
example/** linguist-documentation
```

But there may be a better way.

In [the linguist language override documentation](https://github.com/github-linguist/linguist/blob/master/docs/overrides.md), it mentions that:

> Just like vendored files, Linguist excludes documentation files from your project's language stats. [`documentation.yml`](https://github.com/github-linguist/linguist/blob/master/lib/linguist/documentation.yml) lists common documentation paths and excludes them from the language statistics for your repository.

Checking `documentation.yml` shows there is a regular expression matcher that looks like this:

```yaml
- ^[Ee]xamples/
```

In other words, it would match an `examples` directory.

But the directory is currently named `example`. Not `examples`.

This is an easy fix. Rename the examples directory to be `examples`, not `example`.
